roles/hammerdb/tasks/main.yaml

```diff
@@ -0,0 +1,64 @@
+---
+
+- name: Show template variables
+  debug:
+    msg: "What do we do, init {{hammerdb.db_init}} or benchmark {{hammerdb.db_benchmark}}"
+
+- name: template db creation script
+  k8s: 
+    definition:
+      apiVersion: v1
+      kind: ConfigMap
+      metadata: 
+        name: '{{ meta.name }}-hammerdb-config'
+        namespace: '{{ operator_namespace }}'
+      data: 
+        createdb.tcl: "{{ lookup('template', 'createdb.tcl.j2') }}"
+
+- name: template db workload script
+  k8s:
+    definition:
+      apiVersion: v1
+      kind: ConfigMap
+      metadata:
+        name: '{{ meta.name }}-hammerdb-workload'
+        namespace: '{{ operator_namespace }}'
+      data:
+        workload.tcl: "{{ lookup('template', 'workload.tcl.j2') }}"
+
+- name: start db creation job
+  k8s:
+    state: present
+    definition: "{{ lookup('template', 'db_creation.yml') | from_yaml }}"
+  when: hammerdb.db_init is defined and hammerdb.db_init
+
+- name: wait for db creation job to finish
+  k8s_facts:
+    kind: Job
+    api_version: batch/v1
+    #name: '{{ meta.name }}-hammerdb-hammerdb_creator'
+    name: '{{ meta.name }}-hammerdb-creator'
```

Comment:
following pr #191 any resources created should make use of the `trunc_uuid` var appended to name and label.

Reply:
fixed